storagenode/operator/service.go

```diff
@@ -0,0 +1,72 @@
+// Copyright (C) 2019 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+package operator
+
+import (
+	"context"
+	"github.com/zeebo/errs"
+	"go.uber.org/zap"
+
+	"storj.io/storj/internal/memory"
+	"storj.io/storj/storagenode/bandwidth"
+	"storj.io/storj/storagenode/pieces"
+)
+
+// Service is handling storage node operator related logic
```

Comment:
Why do we need this service when we already have `bandwidth`, `pieces` and other services for this logic.

Reply:
@egonelbre just to separate our business logic from api layer
operator.Service uses bandwidth, pieces, etc to combine all data together